cmd/kops/rollingupdatecluster.go

```diff
@@ -107,6 +129,8 @@ type RollingUpdateOptions struct {
 	// InstanceGroups is the list of instance groups to rolling-update;
 	// if not specified all instance groups will be updated
 	InstanceGroups []string
+
+	Algorithm string
```

Comment:
Maybe a better name for this might be "Strategy" to be consistent with rolling update strategy for Deployments